README.md

```diff
@@ -50,8 +68,8 @@ yarn dev
 
 ### Formatar o código
 
-Antes de dar commit, é recomendado executar o `prettier` (se usarem um editor de texto - e.g. vscode - que já executa automaticamente, não é preciso).
+Antes de fazer um commit, é recomendado executar o `prettier` (se usarem um editor de texto - e.g. vscode - que já executa automaticamente, não é preciso). Devem correr o comando na raiz do _repo_ (`/home/.../resumos-leic`, portanto).
 
 ```bash
-yarn format
+yarn format:content
```

Comment:
This is not necessary anymore, `yarn format` is an alias to that

Reply:
TIL, ty!